TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/DriveSystem.java

```diff
@@ -160,19 +141,43 @@ public void drive(float rightX, float rightY, float leftX, float leftY, boolean
         telemetry.update();
     }
 
-    private void driveToPositionTicks(int ticks) {
-        motorFrontRight.setTargetPosition(motorFrontRight.getCurrentPosition() + ticks);
-        motorFrontLeft.setTargetPosition(motorFrontLeft.getCurrentPosition() + ticks);
-        motorBackRight.setTargetPosition(motorBackRight.getCurrentPosition() + ticks);
-        motorBackLeft.setTargetPosition(motorBackLeft.getCurrentPosition() + ticks);
+    private void driveToPositionTicks(int ticks, Direction direction) {
+        if (direction == Direction.FORWARD) {
+            motorFrontRight.setTargetPosition(motorFrontRight.getCurrentPosition() + ticks);
+            motorFrontLeft.setTargetPosition(motorFrontLeft.getCurrentPosition() + ticks);
+            motorBackRight.setTargetPosition(motorBackRight.getCurrentPosition() + ticks);
+            motorBackLeft.setTargetPosition(motorBackLeft.getCurrentPosition() + ticks);
+        }
+
+        if (direction == Direction.BACKWARD) {
+            motorFrontRight.setTargetPosition(motorFrontRight.getCurrentPosition() - ticks);
+            motorFrontLeft.setTargetPosition(motorFrontLeft.getCurrentPosition() - ticks);
+            motorBackRight.setTargetPosition(motorBackRight.getCurrentPosition() - ticks);
+            motorBackLeft.setTargetPosition(motorBackLeft.getCurrentPosition() - ticks);
+        }
+
+        if (direction == Direction.RIGHT) {
+            motorFrontRight.setTargetPosition(motorFrontRight.getCurrentPosition() - ticks);
+            motorFrontLeft.setTargetPosition(motorFrontLeft.getCurrentPosition() + ticks);
+            motorBackRight.setTargetPosition(motorBackRight.getCurrentPosition() + ticks);
+            motorBackLeft.setTargetPosition(motorBackLeft.getCurrentPosition() - ticks);
+        }
+
+        if (direction == Direction.LEFT) {
+            motorFrontRight.setTargetPosition(motorFrontRight.getCurrentPosition() + ticks);
+            motorFrontLeft.setTargetPosition(motorFrontLeft.getCurrentPosition() - ticks);
+            motorBackRight.setTargetPosition(motorBackRight.getCurrentPosition() - ticks);
+            motorBackLeft.setTargetPosition(motorBackLeft.getCurrentPosition() + ticks);
+        }
 
         setRunMode(DcMotor.RunMode.RUN_TO_POSITION);
-        setMotorPower(0.7);
+        setMotorPower(0.05);
```

Comment:
Can you comment on what exactly you're doing here? (From here to the end of the method...)